src/Views/ProgressView.vala

```diff
@@ -342,8 +212,7 @@ public class ProgressView : AbstractInstallerView {
                     var new_disk = new Distinst.Disk (m.parent_disk);
                     if (new_disk == null) {
                         warning ("could not find physical device: '%s'\n", m.parent_disk);
```

Comment:
Should this be a `critical ()` like the others?

Reply:
Yes, it should.